.github/workflows/windows.yml

```diff
@@ -0,0 +1,131 @@
+        with:
+          fetch-depth: 2
+
+      - name: Setup PHP
+        run: |
+          $env:Path = 'c:\php;' + $env:Path
+          mkdir c:\php && cd c:\php
+          iwr -outf php-7.2.5-Win32-VC15-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php-7.2.5-Win32-VC15-x86.zip
+          7z x php-7.2.5-Win32-VC15-x86.zip -y >nul
+          cd ext
+          iwr -outf php_apcu-5.1.19-7.2-ts-vc15-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php_apcu-5.1.19-7.2-ts-vc15-x86.zip
+          7z x php_apcu-5.1.19-7.2-ts-vc15-x86.zip -y >nul
+          iwr -outf php_redis-5.3.2-7.2-ts-vc15-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php_redis-5.3.2-7.2-ts-vc15-x86.zip
+          7z x php_redis-5.3.2-7.2-ts-vc15-x86.zip -y >nul
+          cd ..
+          Copy php.ini-development php.ini-min
+          "memory_limit=-1" >> php.ini-min
+          "serialize_precision=-1" >> php.ini-min
+          "max_execution_time=1200" >> php.ini-min
+          "post_max_size=2047M" >> php.ini-min
+          "upload_max_filesize=2047M" >> php.ini-min
+          "date.timezone=`"America/Los_Angeles`"" >> php.ini-min
+          "extension_dir=ext" >> php.ini-min
+          "extension=php_xsl.dll" >> php.ini-min
+          "extension=php_mbstring.dll" >> php.ini-min
+          Copy php.ini-min php.ini-max
+          "zend_extension=php_opcache.dll" >> php.ini-max
+          "opcache.enable_cli=1" >> php.ini-max
+          "extension=php_openssl.dll" >> php.ini-max
+          "extension=php_apcu.dll" >> php.ini-max
+          "extension=php_redis.dll" >> php.ini-max
+          "apc.enable_cli=1" >> php.ini-max
+          "extension=php_intl.dll" >> php.ini-max
+          "extension=php_fileinfo.dll" >> php.ini-max
+          "extension=php_pdo_sqlite.dll" >> php.ini-max
+          "extension=php_curl.dll" >> php.ini-max
+          "extension=php_sodium.dll" >> php.ini-max
+          Copy php.ini-max php.ini
+          cd D:\a\symfony\symfony
+          iwr -outf composer.phar https://getcomposer.org/download/latest-stable/composer.phar
+          mkdir %APPDATA%\Composer && Copy .github\composer-config.json %APPDATA%\Composer\config.json
+
+      - name: Install dependencies
+        id: setup
+        shell: cmd
+        run: |
+          SET PATH=c:\php;%PATH%
+          mkdir %APPDATA%\Composer && copy /Y .github\composer-config.json %APPDATA%\Composer\config.json
+          FOR /F "tokens=* USEBACKQ" %%F IN (`bash -c "grep ' VERSION = ' src/Symfony/Component/HttpKernel/Kernel.php | grep -o '[0-9][0-9]*\.[0-9]'"`) DO (SET SYMFONY_VERSION=%%F)
+          php .github/build-packages.php HEAD^ %SYMFONY_VERSION% src\Symfony\Bridge\PhpUnit
+          SET COMPOSER_ROOT_VERSION=%SYMFONY_VERSION%.x-dev
+
+          echo ::group::composer update
+          php composer.phar update --no-progress --ansi
+          echo ::endgroup::
+
+          echo ::group::phpunit install
```

Comment:
instead of those manual grouping, maybe we could just use separate steps in the job (which are collapsed by default in the GitHub UI so having many steps is not an issue)

Reply:
That's a good idea, but maybe we do that in a follow-up PR as this was (as I guess) mostly taken from the existing jobs where we do it this way?